integration_test/Bindings/Python/dialects/om.py

```diff
@@ -82,6 +85,8 @@
 print(obj.child.foo)
 # CHECK: ('Root', 'x')
 print(obj.reference)
+# CHECK: 14
+print(obj.tuple[1])
```

Comment:
Can we unpack these tuples via `a, b, c = obj.tuple`?

Reply:
That should work because the tuple has an `__iter__` method :thinking: